components/eam/tools/mkatmsrffile/mkatmsrffile.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/env python3
+#---------------------------------------------------------------------------------------------------
+'''
+This is a replacement for the legacy gen_domain tool created for CESM. 
```

Comment:
This comment seems to be copied from one for a new gen_domain tool. Similarly the `source_code_meta = 'generate_domain_E3SM.py'`

Reply:
fixed.